core/templates/dev/head/components/summary_tile/ExplorationSummaryTileDirective.js

```diff
@@ -130,7 +131,17 @@ oppia.directive('explorationSummaryTile', [
             } else {
               var result = '/explore/' + $scope.getExplorationId();
               if ($scope.getCollectionId()) {
-                result += ('?collection_id=' + $scope.getCollectionId());
+                result = UrlService.addParams(result, 'collection_id',
```

Comment:
In general, if a set of args within parentheses spans multiple lines, break after the first '('. Ditto below.

Reply:
Done